hazelcast/serialization/compact.py

```diff
@@ -43,7 +43,11 @@ def __init__(self, compact_serializers: typing.List[CompactSerializer]):
 
     def write(self, out: ObjectDataOutput, obj: typing.Any) -> None:
         clazz = type(obj)
-        serializer = self._type_to_serializer[clazz]
+        serializer = self._type_to_serializer.get(clazz)
+        if serializer is None:
+            raise HazelcastSerializationError(
+                "No serializer is registered for class {}.".format(type(obj).__name__)
```

Comment:
```suggestion
                f"No serializer is registered for class {clazz}."
```

Let's convert it to this to make use of f-strings and printing the class name with the package, which will be more useful for debugging

Reply:
Refactored at https://github.com/hazelcast/hazelcast-python-client/pull/574/commits/3e052df6aaaf866e1f977a76832ca4c1577e85f4. Using `clazz` without `__name__` gives output as follows: `No serializer is registered for class/constructor <class 'tests.integration.backward_compatible.serialization.compact_test.Nested'>.`. </br></br> I added `__name__` to sync the output with Node.js client, should we use the long output, wdyt @mdumandag?

Reply:
I believe we should use the long output, because it is easier to understand what class we are talking about, when it also has the package name.